src/shogun/mathematics/SFMT.cpp

```diff
@@ -0,0 +1,158 @@
+#include <shogun/lib/external/SFMT/SFMT.h>
+#include <shogun/lib/external/dSFMT/dSFMT.h>
+#include <shogun/mathematics/SFMT.h>
+
+using namespace shogun;
+
+template <typename T>
+SFMT<T>::SFMT(uint32_t seed) : m_seed(seed)
```

Comment:
why??? we dont want this... no need for this lib

Reply:
I want to get rid of CRandom
porting this well allow us to use the SIMD instructions it provides, and get rid of CRandom until there is an alternative

Reply:
no you want to get rid of CRandom and *SFMT... meaning leave them there as they are, dont add more stuff to it. but start porting stuff that is using CRandom and once all CRandom use-cases are ported we can drop the whole thing

Reply:
and i thought we've agreed that even though *SFMT is good, it is unmaintained and we are certainly not going to maintain it... so we wanna get rid of it, even if it gives you SIMD random.

Reply:
I misunderstood you on that point. I thought we couldn't get rid of it yet because of the SIMD stuff.